SettingURLSession.playground/Contents.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import CryptoKit
+
+class CreatureURL {
+
+    func md5Hash(_ source: String) -> String {
+        return Insecure.MD5.hash(data: source.data(using: .utf8)!).map { String(format: "%02hhx", $0) }.joined()
+    }
+
+    func buildURL(scheme: String, host: String, path: String) -> URL? {
+
+        var components: URLComponents = {
+            var components = URLComponents()
+            components.scheme = scheme
+            components.host = host
+            components.path = path
+            return components
+        }()
+        return components.url
+    }
+}
+
+class CreatureURLMarvel: CreatureURL {
+
+    override func buildURL(scheme: String, host: String, path: String) -> URL? {
+
+        let publicKey = "8ff288aa4d9bd9b25d051aea9585e041"
+        let privateKey = "745eeeff855e804c98b080fd6570b1b2b8553550"
+        let ts = NSDate().timeIntervalSince1970.description
```

Comment:
Тут можно и Swift'овым Date() воспользоваться
```suggestion
        let ts = Date().timeIntervalSince1970.description
```